transcom/mymove
# INT-B-21610

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1027078&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

### How to test

- Given a move with destination SIT (To recreate, make a new move -> approve -> request destination SIT, don't approve SIT yet)
- As TOO, approve ONLY DDSFSC (Should recreate with all other service items besides DDASIT too, but DDSFSC is confirmed)
- As Prime, update shipment destination address
- Approve as TOO if the new address was greater than 50 miles away, if not skip this step
- As Prime, create a payment request for the service item you approved as TOO
- No error

Try creating a payment request with different combinations of approved/rejected service items.
